contracts/SimpleSaleContract.sol

```diff
@@ -6,6 +6,13 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "./MeetupERC20Token.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
+import "hardhat/console.sol";
+
+interface Weth is IERC20 {
```

Comment:
For clarity, it is a good practice to store interfaces in a separate directory under "contracts" directory.